internal/guides/generated/producer/substitutions.go

```diff
@@ -156,7 +156,141 @@ substitution.
       spec:
         containers:
         - name: nginx
-          image: nginx:1.8.1 # {"$ref":"#/definitions/io.k8s.cli.substitutions.image-value"}
+          image: nginx:1.8.1 # {"$kpt-set":"image-value"}
+
+## Nested Substitutions
```

Comment:
Why is this part of this PR ? Isn't it already merged ?

Reply:
I checked the master, it seams that it's not updated in the generated doc, it's possible that last person update the doc but forgot to run the command to update the generated doc, so when I run, it update that part as well.
https://github.com/GoogleContainerTools/kpt/blob/master/internal/guides/generated/consumer/substitute.go